zhusuan/model/base.py

```diff
@@ -300,6 +300,18 @@ def _add_stochastic_tensor(self, s_tensor):
         else:
             self._stochastic_tensors[s_tensor.name] = s_tensor
 
+    def get(self, name_or_names):
+        """
+        Get the `StochasticTensor` in the network by their names.
+
+        :param name_or_names: A string or a list of strings. Names of
+            `StochasticTensor` s in the network.
+        """
+        if isinstance(name_or_names, (tuple, list)):
+            return [self._stochastic_tensors[name] for name in name_or_names]
+        else:
+            return self._stochastic_tensors[name_or_names]
+
     def outputs(self, name_or_names):
```

Comment:
Can you open another PR that adds name checks to all functions of BayesianNet? raise an error like there is no StochasticTensor named xxx in the BayesianNet.